src/parser/jobs/whm/modules/Overheal.tsx

```diff
@@ -9,25 +9,28 @@ export class Overheal extends CoreOverheal {
 	override displaySuggestion = true
 
 	override trackedHealCategories = [
+		{
+			name: 'Assize',
+			trackedHealIds: [this.data.actions.ASSIZE.id],
+			ignore: true,
+		},
 		{
 			name: <Trans id="whm.overheal.hot.name">Healing Over Time</Trans>,
-			color: '#157f1f',
+			color: SuggestedColors[1],
 			trackedHealIds: [
 				this.data.statuses.REGEN.id,
 				this.data.actions.MEDICA_II.id,
 				this.data.statuses.MEDICA_II.id,
 				this.data.actions.ASYLUM.id,
 				this.data.statuses.ASYLUM.id,
+				this.data.actions.MEDICA_III.id,
```

Comment:
_Consider:_ I realize this is keeping the existing behavior, but is it really correct to treat the "direct" heal portion of Medica 2/3 as HOT overheal?  The implication I understand for the overheal bucketing is that direct overheal is more severe than HOT overheal, so would it make more sense to separate the direct heal component of Medica 2/3?

Reply:
Good catch, direct healing portion of Medica 2/3 are now considered normal overheal instead of HoT.